docs/docs/tutorials/adding_annotations.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: Adding the grpc-gateway annotations to an existing protobuf file
+parent: Tutorials
+nav_order: 6
+---
+
+## Adding the grpc-gateway annotations to an existing protobuf file
+
+Now that we've got a working Go gRPC server, we need to add the grpc-gateway annotations:
+
+```proto
+syntax = "proto3";
+
+package helloworld;
+
+import "google/api/annotations.proto";
+
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {
+        option (google.api.http) = {
+        post: "/v1/example/echo"
+        body: "*"
+    };
+  }
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+### Generating the grpc-gateway stubs
+
+Now that we've got the grpc-gateway annotations added to the proto file, we need to use the grpc-gateway generator to generate the stubs.
+
+Before we can do that, we need to copy some dependencies into our protofile structure. Copy the `third_party/googleapis` folder from the grpc-gateway repository to your local protofile structure. It should look like this afterwards:
+
+''' (backticks)
+proto/
+helloworld/
+hello_world.proto
+google/
+api/
+http.proto
+annotations.proto
+''' (backticks)
+
+#### Using buf
+
+[Using buf](generating_stubs/using_buf.md)
+
+#### Using protoc
+
+[Using protoc](generating_stubs/using_protoc.md)
+
+Start the greeter_server service first, and then start the gateway. Then gateway connects to greeter_server and establishes http monitoring.
```

Comment:
Remove this, this doesn't make any sense. This is where we need to show the changes that need to be made to main.go now that we've got a generated gateway stub. I think it might be best not to refer to the boilerplate here, we should show the changes that are necessary (e.g. the RegisterBlaFromBla function call and subsequent serving of the HTTP server).